src/Responses/AuthSuccess.php

```diff
@@ -1,9 +1,12 @@
 <?php
 
-namespace Saritasa\Laravel\Controllers\Responses;
+namespace Saritasa\LaravelControllers\Responses;
 
 use Saritasa\Transformers\DtoModel;
 
+/**
+ * Auth success response message.
+ */
 class AuthSuccess extends DtoModel
```

Comment:
Should it really extend DtoModel and not Dto class? I know, that it is not your changes, but let's just fix it